Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -834,9 +854,8 @@ static void gdb_init_generic(int domain, const sockaddr* server_addr, socklen_t
                              sockaddr* client_addr, socklen_t* client_addrlen)
 {
 #ifdef _WIN32
-  WSAStartup(MAKEWORD(2, 2), &InitData);
+  s_wsa = std::make_unique<WSA_RAII>();
```

Comment:
With the SocketContext thing mentioned above, you would be able to remove the ifdef here.